.changelog/v0.12.0/summary.md

```diff
@@ -0,0 +1,13 @@
+This release notably brings compatibility with Cosmos SDK 0.45 and IBC v3.0.0-rc.0,
+as well as support for non-standard ports in the channel handshake.
+It also contains a fix for a bug where `SendPacket` events were duplicated when emitted at EndBlock,
+and fixes another bug where Hermes would clear packet at startup even when `clear_on_start = false`.
+Additionally, a new CLI command `clear packets` has been added for clearing packets in both direction on a given channel.
```

Comment:
This remark about `clear packets` should probably be in the "operators" section.

Reply:
e4e5ec52